lib/screens/explore_screens/vote_screens/proposals/components/proposal_card.dart

```diff
@@ -116,19 +116,20 @@ class _ProposalCardState extends State<ProposalCard> with AutomaticKeepAliveClie
                                       ),
                                     ),
                                   ),
-                                  LayoutBuilder(
-                                    builder: (_, constrains) {
-                                      // If voice needed > total show 100% else show percent.
-                                      // triangle position - triangle middle width - left margin
-                                      final leftPadding = widget.proposal.total < widget.proposal.voiceNeeded
-                                          ? constrains.maxWidth - 6 - 16
-                                          : constrains.maxWidth * widget.proposal.voiceNeededBarPercent - 6 - 16;
-                                      return Padding(
-                                        padding: EdgeInsets.only(left: leftPadding, top: 20),
-                                        child: const CustomPaint(size: Size(12, 8), painter: TrianglePassValue()),
-                                      );
-                                    },
-                                  ),
+                                  if (widget.proposal.stage.isNotEmpty)
+                                    LayoutBuilder(
+                                      builder: (_, constrains) {
+                                        // If voice needed > total show 100% else show percent.
+                                        // triangle position - triangle middle width - left margin
+                                        final leftPadding = widget.proposal.total < widget.proposal.voiceNeeded
+                                            ? constrains.maxWidth - 6 - 16
+                                            : constrains.maxWidth * widget.proposal.voiceNeededBarPercent - 6 - 16;
+                                        return Padding(
+                                          padding: EdgeInsets.only(left: leftPadding, top: 20),
+                                          child: const CustomPaint(size: Size(12, 8), painter: TrianglePassValue()),
+                                        );
+                                      },
+                                    ),
```

Comment:
The other day I asked you for the quorum, It was to know in the case of the referendums the `voice needed` to say that it has been passed. 
Since we don't have a referendum UI I assumed that this progress bar should always be displayed with the triangle indicating the voice needed.

![image](https://user-images.githubusercontent.com/42857405/132769333-e099c33b-4a30-4b15-9e42-ba6ab0ec96c3.png)

Then can the quorum value be considered as the `voice needed`?

Reply:
it's % of citizens so you would need to pull the number of citizens from somewhere.

Reply:
I thought that % was the quorum value.

Reply:
Ok now Sorin said it's the 90% point we need to pass anything - so that's easy, if it's just 90% of total votes there. Just place it at 0.9x offset and be done with it ;)